### PR TITLE
Add RZ.20

### PR DESCRIPTION
**Pull request by Capkirk123**
Add the RZ.20, a hydrolox upper stage engine created by Rolls-Royce

### CODE REVIEW

File: GameData/ROEngines/PartConfigs/RZ20_Tantares.cfg [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+		name = ModuleB9DisableTransform
+		transform = base_s0p5
+	}
+
+	MODULE
+	{
+		name = ModuleB9DisableTransform
+		transform = base_s1
+	}
+}
```
Comment (DRVeyl) [inline GameData/ROEngines/PartConfigs/RZ20_Tantares.cfg:110]: Add newline before EOL

**Review by DRVeyl — CHANGES_REQUESTED** [review]
Why are the model files and texture files empty?

**Comment by Capkirk123** [comment]
Strange bug with Github desktop I guess. Renaming everything and then changing it back got it to upload everything properly.

File: GameData/ROEngines/PartConfigs/RZ20_Tantares.cfg [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+	{
+		model = ROEngines/Assets/Tantares/black_anvil_engine_s1_1
+	}
+
+	scale = 1
+	rescaleFactor = 1.6393
+
+	node_stack_top	  = 0.0,  0.5642, 0.0, 0.0, 1.0, 0.0, 1
+	node_stack_bottom = 0.0, -0.625, 0.0, 0.0, -1.0, 0.0, 1
+	node_attach		  = 0.0,  0.5642, 0.0, 0.0,	1.0, .0
```
Comment (DRVeyl) [inline GameData/ROEngines/PartConfigs/RZ20_Tantares.cfg:24]: Indenting / last entry should be 0.0?

**Review by DRVeyl — APPROVED** [review]

**Pull request merged**